### ISSUE TITLE
Handling class imbalance while fine-tuning on TACRED

### ISSUE THREAD

**Issue by aakashb95** [studio-ousia/luke#111]
Thank you for such a wonderful repo. I have been reading your code for relation extraction task. 
I am fine-tuning luke-base and found a dataset at hugging face hub which looks similar to TACRED: https://huggingface.co/datasets/xiaobendanyn/tacred
On inspecting the data, I see the following counts:
![image](https://user-images.githubusercontent.com/48802744/154406500-db8a609a-db09-4511-af67-25f2b1df9cea.png)

How should I deal with class imbalance as 79.6% labels here are no_relation labels. Could you please point me to the line of code in your repo which handles this or any other advice you might have? 

P.S: I have tried to use FocalLoss and weighted CrossEntropy loss but to not much success.

**Comment by ikuyamada** [comment]
Hi @aakashb95,
We did not address the class imbalance issue in TACRED in our experiments, and simply used cross entropy loss to solve the task. However, techniques such as up- or down-sampling may be effective to improve the results.
